eng/pipelines/templates/variables/globals.yml

```diff
@@ -11,4 +11,12 @@ variables:
   REPOROOT: $(Build.SourcesDirectory)
   WINDOWS_OUTPUTROOT: $(REPOROOT)\out
   WindowsContainerImage: 'onebranch.azurecr.io/windows/ltsc2019/vse2022:latest'
-  GDN_SUPPRESS_FORKED_BUILD_WARNING: true
\ No newline at end of file
+  GDN_SUPPRESS_FORKED_BUILD_WARNING: true
+  # the following variables are referenced from the test matrix files in the repo
+  # these variables will all move one version up when a new version is released
+  PYVERSION_N0: 3.13
```

Comment:
Looking at N* in the various templates makes it kind of difficult to reason about. Are there any other naming conventions like Latest, Current, LTS, etc that might make more sense in those contexts?

Reply:
There aren't.

Reply:
@scbedd I see the following from here: https://devguide.python.org/versions/

EOL, Security, Bugfix, Feature

So maybe like below, although it will probably end up mismatching as the version schedule moves up before we do.

```
PYTHON_VERSION_FEATURE
PYTHON_VERSION_BUGFIX
PYTHON_VERSION_SECURITY_LATEST
PYTHON_VERSION_SECURITY_PREVIOUS
PYTHON_VERSION_SECURITY_OLDEST
PYTHON_VERSION_SECURITY_NO_REALLY_OLDEST_I_HATE_THIS
```